server/controllers/course-controller.js

```diff
@@ -0,0 +1,37 @@
+const knex = require("./../db");
+
+// Get the list of lectures scheduled for a course
+exports.getScheduledLectures = async (req, res) => {
+  const courseId = req.params.courseid;
+  knex
+    .select("id", "name", "course", "lecturer", "start", "end", "capacity")
+    .from("lecture")
+    .where("course", courseId)
```

Comment:
It would be nice to have the name and surname of the lecturer, since just the id is not very useful.

Reply:
Resolved